src/components/core/pivotTable/tests/aggregationsMenuHelper.spec.ts

```diff
@@ -4,6 +4,45 @@ import { FIELD_TYPE_ATTRIBUTE, FIELD_TYPE_MEASURE } from '../../../../helpers/ag
 import aggregationsMenuHelper from '../aggregationsMenuHelper';
 import { IColumnTotal } from '../AggregationsMenu';
 
+describe('getTotalTypesAppliedOnAllMeasures', () => {
+    const totals: AFM.ITotalItem[] = [
+        {
+            type: 'sum',
+            attributeIdentifier: 'a1',
+            measureIdentifier: 'm1'
+        },
+        {
+            type: 'sum',
+            attributeIdentifier: 'a1',
+            measureIdentifier: 'm2'
+        },
+        {
+            type: 'max',
+            attributeIdentifier: 'a1',
+            measureIdentifier: 'm1'
+        },
+        {
+            type: 'min',
+            attributeIdentifier: 'a1',
+            measureIdentifier: 'm2'
+        }
+    ];
+
+    it('should return sum as only defined type for both m1 and m2', () => {
+        const measures = ['m1', 'm2'];
+
+        expect(aggregationsMenuHelper.getTotalTypesAppliedOnAllMeasures(totals, measures))
+            .toEqual(['sum']);
+    });
+
+    it('should return sum, max types as they are defined for m1', () => {
```

Comment:
This seems a little bit misleading with tested function name `getTotalTypesAppliedOnAllMeasures` as totals fixture contains also definitions for `m2`. Maybe split it to two different fixtures?

Reply:
renamed - removed "All" so it makes more sense